src/Employer/UnitTests/Employer.Web/UtilityTests/GetAuthorisedApplicationReviewAsyncTests.cs

```diff
@@ -20,33 +20,41 @@ public class GetAuthorisedApplicationReviewAsyncTests
             string requestedEmployerAccountId, bool shouldAllow)
         {
             var applicationReviewId = Guid.NewGuid();
+            var vacancyId = Guid.NewGuid(); 
 
             var client = new Mock<IRecruitVacancyClient>();
             client.Setup(c => c.GetApplicationReviewAsync(applicationReviewId)).Returns(Task.FromResult(
                 new ApplicationReview
                 {
-                    Id = applicationReviewId,
-                    EmployerAccountId = applicationReviewEmployerAccountId,
+                    Id = applicationReviewId,                    
                     VacancyReference = 1000000001
                 }));
 
+            client.Setup(c=>c.GetVacancyAsync(vacancyId)).Returns(Task.FromResult(
+                new Vacancy() {
+                    EmployerAccountId = applicationReviewEmployerAccountId,
+                    VacancyReference = 1000000001,
+                    Id = vacancyId
+                }));
+
             var rm = new ApplicationReviewRouteModel
             {
                 EmployerAccountId = requestedEmployerAccountId,
-                ApplicationReviewId = applicationReviewId
+                ApplicationReviewId = applicationReviewId,
+                VacancyId = vacancyId
             };
 
             Func<Task<ApplicationReview>> act = () => Utility.GetAuthorisedApplicationReviewAsync(client.Object, rm);
 
             if (shouldAllow)
             {
                 var applicationReview = act().Result;
-                applicationReview.EmployerAccountId.Should().Be(requestedEmployerAccountId);
+                applicationReview.Should().NotBeNull();                
             }
             else
             {
                 var ex = Assert.ThrowsAsync<AuthorisationException>(act);
-                ex.Result.Message.Should().Be($"The employer account 'ANOTHER EMPLOYER ACCOUNT ID' cannot access employer account 'EMPLOYER ACCOUNT ID' application '{applicationReviewId}' for vacancy '1000000001'.");
+                ex.Result.Message.Should().Be($"The employer account 'ANOTHER EMPLOYER ACCOUNT ID' cannot access employer account 'EMPLOYER ACCOUNT ID' vacancy ' ({vacancyId})'.");                
```

Comment:
It would be good include the `applicationReviewId` in the error message.  Maybe we should try/catch around `CheckAuthorisedAccess` and throw our own exception to extend the message with the `applicationReviewId`

Reply:
ok!